packages/warriorjs-core/src/Unit.js

```diff
@@ -12,17 +12,17 @@ class Unit {
    * @param {number} reward The number of points to reward when killed.
    * @param {boolean} captive Whether the unit is a captive or not.
    */
-  constructor(name, character, maxHealth, reward = null, captive = false) {
+  constructor(name, character, maxHealth, reward = null, hostile = true) {
```

Comment:
Let's pass a `bound` parameter as the last parameter here. See update in #61 for details.